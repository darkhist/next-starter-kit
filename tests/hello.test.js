import { shallow } from 'enzyme';
import Home from '../pages/index';

describe('Home', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Home />);
  });

  it('should render without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
