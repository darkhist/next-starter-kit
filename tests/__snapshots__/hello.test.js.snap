// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render without crashing 1`] = `ShallowWrapper {}`;
